### PR TITLE
add timeout to influx db

### PR DESCRIPTION
**Pull request by romsharon98**
passing extra to influxDB.
close [this](https://apache-airflow.slack.com/archives/CCQ7EGB1P/p1719314181265999) thread in slack: 

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by potiuk** [comment]
Hmm. I guess we should add (or modify) existing test to test for it. I find it highly suspicious when such changes pass without any test modifications - it makes me think some tests are missing.

File: docs/apache-airflow-providers-influxdb/connections/influxdb.rst [hunk 1]
```diff
@@ -31,17 +31,12 @@ Extra (required)
     Specify the extra parameters (as json dictionary) that can be used in InfluxDB
     connection.
 
-    ``token``: (required) `Create token <https://docs.influxdata.com/influxdb/cloud/security/tokens/create-token/>`_
-    using the influxdb cli or UI
-
-    ``org_name``: (required) `Create org <https://docs.influxdata.com/influxdb/cloud/reference/cli/influx/org/create/>`_
```
Comment (eladkal) [inline docs/apache-airflow-providers-influxdb/connections/influxdb.rst:37]: If token and org name are required lets customize the Ui form for to have them. Its much nicer than passing them as kwargs

**Comment by romsharon98** [comment]
> Hmm. I guess we should add (or modify) existing test to test for it. I find it highly suspicious when such changes pass without any test modifications - it makes me think some tests are missing.

Yes, you were right. The tests didn't check much (or at all). I've updated them.

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Cool. Thanks!

File: docs/apache-airflow-providers-influxdb/connections/influxdb.rst (hunk 1 above)
Comment (eladkal) [reply docs/apache-airflow-providers-influxdb/connections/influxdb.rst:37]: @romsharon98 i think we shouldnt remove this but just mention its not defined via extra. 
This is what we do in other providers as well 
https://airflow.apache.org/docs/apache-airflow-providers-google/stable/connections/bigquery.html#configuring-the-connection

**Pull request merged**